config/env_sample.hjson

```diff
@@ -27,14 +27,14 @@
         "REPORT_ONLY": true,
         # You should set your domain as the DEFAULT_SRC
         "DEFAULT_SRC": ["'self'","example.edu"],
-        "SCRIPT_SRC": ["'self'", "'unsafe-inline'", "'unsafe-eval'", "www.google-analytics.com"],
-        "IMG_SRC": ["'self'", "data:", "www.google-analytics.com"],
+        "SCRIPT_SRC": ["'self'", "'unsafe-inline'", "*.googletagmanager.com"],
```

Comment:
Unfortunately, looks like the `pylti1p3`'s `script` tags are preventing us from removing `unsafe-inline` entirely here. It's no longer necessary for Google Analytics since we're passing the nonce.